addons/editor-square-inputs/addon.json

```
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://raw.githubusercontent.com/ScratchAddons/manifest-schema/dist/schema.json",
+  "name": "Square block text and color inputs",
+  "description": "Makes text and/or color inputs on blocks rectangular instead of round, like in Scratch 2.0.",
```

Just and should be okay here since it's the default.
```suggestion
  "description": "Makes text and color inputs on blocks rectangular instead of round, like in Scratch 2.0.",
```